src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -41,8 +39,6 @@ const useLocalDatabaseHook = (withMigration = false) => {
   const {channelInfo, channelList, channelMember, chat, user} = databaseListener;
 
   const initDb = async () => {
-    const isEnteringApp = Boolean(initialStartup?.id);
-
     const db = await LocalDatabase.getDBConnection();
     const shouldDbMigrated = await LocalDatabaseMigration.shouldDbMigrated();
     let scopedMigrationStatus = migrationStatus;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `isEnteringApp` variable might have an impact on the logic of your database initialization. Make sure that the migration process is still correctly triggered without this condition.



<!-- This is an auto-generated comment by OSS CodeRabbit -->